File: 0001_0100/0028/0028.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    x1, y1, x2, y2, xa, ya = map(int, f.read().split())
    a = (2 * x1 - xa, ya) if x1 == x2 else (xa, 2 * y1 - ya)
    q.write(str(a[0]) + ' ' + str(a[1]))

File: 0001_0100/0028/0028.go
package main

import (
	"fmt"
	"os"
)

func main() {
	f, _ := os.Open("input.txt")
	q, _ := os.Create("output.txt")
	var x1, y1, x2, y2, x, y int
	fmt.Fscan(f, &x1, &y1, &x2, &y2, &x, &y)
	if x1 == x2 {
		x = 2*x1 - x
	} else {
		y = 2*y1 - y
	}
	fmt.Fprint(q, x, " ", y)
}
